src/browser/fetcher.rs

```diff
@@ -247,7 +247,7 @@ fn archive_name<R: AsRef<str>>(_revision: R) -> Result<&'static str, Error> {
     #[cfg(windows)]
     {
         // Windows archive name changed at r591479.
-        if revision.as_ref().parse::<u32>()? > 591_479 {
+        if _revision.as_ref().parse::<u32>()? > 591_479 {
```

Comment:
Underscored variables should not actually be used. I Suggest to rename the input variable to `revision` and do `drop(revision); Ok(..)` in the linux and macos paths.